README.md

```diff
@@ -181,7 +181,7 @@ CARGO_BUILD_PIPELINING=true cargo build
 
 ### Getting the rest of the system working
 
-To get other components ([TiDB](https://github.com/pingcap/tidb) and [PD](https://github.com/pingcap/pd)) working, we suggest you follow the [development guide](https://github.com/pingcap/docs/blob/master/dev-guide/development.md), because you need the `pd-server` at least to work alongside `tikv-server` for integration level testing.
+To get other components ([TiDB](https://github.com/pingcap/tidb) and [PD](https://github.com/pingcap/pd)) working, we suggest you follow the [development guide](https://github.com/pingcap/docs/blob/master/v2.1-legacy/dev-guide/development.md), because you need the `pd-server` at least to work alongside `tikv-server` for integration level testing.
```

Comment:
why do we reference to 2.1 documentation instead of dev?

Reply:
because this file does not exist in dev after refactoring.

Reply:
Should we add it back to documentation? It just looks wired for a documentation that does not include contents about how to develop.

Reply:
I will discuss with @lilin90 tomorrow. I think the current documentation is more user-oriented and task-based. During the process, we had to ditch some topics because they didn't quite fit in the content structure.

Reply:
Both me and Lilian think we can add the link of those guide in Readme in the TiDB Repo.
What do you think? If it sounds fine, let's merge this PR for now.